packages/preview-middleware/src/base/flp.ts

```diff
@@ -0,0 +1,188 @@
+import type { ReaderCollection } from '@ui5/fs';
```

Comment:
Why would we want that?

Reply:
For testing with a different theme or for adding additional libs that are not part of the `manifest.json`. At least we need the check for the `fiori-tools-rta-mode`, because this parameter is essential for starting in the respective UIAdaption Mode.

Reply:
works now, thx